frontend/apps/app/app/lib/vectorstore/syncSchemaVectorStore.ts

```diff
@@ -111,13 +38,13 @@ function validateEnvironmentVariables(): boolean {
  * Synchronizes schema data with vector store
  * Can be called directly from API routes
  * @param schemaData The schema data to synchronize
- * @param projectId Optional project ID to get organization ID
+ * @param organizationId Organization ID for vector store
  * @param forceUpdate Force update even if schema hasn't changed
  * @returns True if synchronization was performed, false otherwise
  */
 export async function syncSchemaVectorStore(
   schemaData: Schema,
-  projectId: string,
+  organizationId: string,
```

Comment:
The reason why projectId was needed for SessionDetailPage, etc. was to pass to this function, and also what was really needed for this function was `organizationId`. 

Therefore, modified to accept organizationId and pass organizationId throughout.